sys/shell/commands/sc_netif.c

```diff
@@ -270,6 +270,10 @@ static void _netif_list(kernel_pid_t dev)
             printf("6LO  ");
         }
         linebreak = true;
+
+        printf("Link type: %s  ", (entry->flags & GNRC_IPV6_NETIF_FLAGS_IS_WIRED) ?
+                "wired" : "wireless");
+        linebreak= true;
```

Comment:
Can we put this in an extra line? Though it is a flag internally, the way you print it does not show it as a flag.